templates/fr/docs/routing/_hosts.md

```diff
@@ -0,0 +1,33 @@
+## Hôtes
+
+<%= sinceVersion("0.18.2") %>
+
+Buffalo prennent également en charge le regroupement des points de terminaison par hôte. L'hôte crée un nouveau groupe qui correspond au domaine transmis. Ceci est utile pour créer des groupes de points de terminaison pour différents domaines ou sous-domaines.
```

Comment:
Host creates a new group that matches the domain passed. This is useful for creating groups of end-points for different domains or subdomains.
```suggestion
Buffalo supporte également le regroupement d'APIs par domaine. La méthode `Host` crée un nouveau groupe pour le nom de domaine passé en paramètre, ce qui peut être une méthode pratique pour regrouper des APIs appartenant au même domaine (voire gérer plusieurs sous-domaines sur une même application).
```

Reply:
Updated in https://github.com/gobuffalo/docs/pull/605/commits/d4b35fe13f94431f3aa51537735d4e1fd5d59348